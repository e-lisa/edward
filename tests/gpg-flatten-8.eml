From: No One <noone@localhost>
MIME-Version: 1.0
To: No One <noone@localhost>
Subject: =?UTF-8?B?44GT44KT44Gr44Gh44Gv44CB44GK5YWD5rCX44Gn44GZ44GL77yf?=
Content-Type: multipart/encrypted;
 protocol="application/pgp-encrypted";
 boundary="x2iAciceo0NrKsl4uFVFGTgJ6k0ocg8FW"

This is an OpenPGP/MIME encrypted message (RFC 4880 and 3156)
--x2iAciceo0NrKsl4uFVFGTgJ6k0ocg8FW
Content-Type: application/pgp-encrypted
Content-Description: PGP/MIME version identification

Version: 1

--x2iAciceo0NrKsl4uFVFGTgJ6k0ocg8FW
Content-Type: application/octet-stream; name="encrypted.asc"
Content-Description: OpenPGP encrypted message
Content-Disposition: inline; filename="encrypted.asc"

-----BEGIN PGP MESSAGE-----
Version: GnuPG v1

hIwDWouAYlp2iv8BA/0SDY0S389Y8+g3t5lkGBnaMC5r1m98cHA1hZwjBHyoZXot
yIMqwvceeeZfqTDC3KSwzDap6EAH63Z22iJZK8K8P9tYjOJ48EagMoIddm1bifC3
iNNwlN/ryWeR+kc1A9nXD/sMaZXW7T+5H1FWNG26jkEkK5zMfUsVWTJrY3a1PNLq
ATpYfoAonBz6dVWJiaVBMpZ8WZnoG/r6fhHHThM/7flnNvIjfNmZRokIysTqwtk4
NpTT9EzgHb1Jrtj/X7bUiAKHBMZcisib4y9VvX866fToU2/tzTs12BQV6IopX+Ah
tDoBcXz18F/z749BnJ3PHx6mX0dQaqeHe/0kF4ZQKaCWsLxxT0FeVmFcEhzu+9sv
whMr/odBQF1tKMMiOqDdEtErKvWhhvHaVLgbFQHonLWl8U/8PPFCntKM0NCs5UuU
ahLGuAvWLiaXzjGSmHj88gAFbgAtub5RregR81HrLevXRZGd7vok+Epv4p+QwaZV
6UqPKlamFnZFQnVQEG7bY/237iF9YJBCUprpFw1kgIeyyLKPNwURTujkXVuRm1+u
URrbo+JZJ2P8SXTVdVLZ+Cxq/ld+xOQRYBHKhrOpAY4Cxe2MV+mFSqrXyOWYpJYk
fkcWS4andIWa9p6UYZoOc9jqKaucDkTwt+HwzR+q4eKehL5g32WNtmzpN4q5yChS
qM0FZgQBXyDXZOGfw/+32LtlJejbGRoi08ZredA3E3RjcD/U1Ag7s4oo4yXwWfAD
Fac9hOzo3F9RQQoNCCFLHzo+vCdkgkDS71lhyt/TiJudiho1c34y2jStkK3YHE29
N/YZc/VvXP5CcpYIPE20r17PBB8dp0gJUZf3kL7hSaLXysmCpI8nZZsBdFlx4Z3Q
CXZB3OEinTuvTb5JcewTVy3/VQr78B325xJkkR9jzbMWw9JrqRmKOeNvyYURPo0a
ETYCbZNaPAY3IwQPIm6cuz/XIq6hKOMoWSHbyhw7X82OAJU//SDgJXYirpGPA33F
LZuRHk5pPGiNNj23Z4p0579S4SSAQ83n2dLo2+K6NbfUvJJxyZW6Jm9/g0yi9+3Y
hg4cGk6XrXNw29Ye023bbO+HRI0wp4CRqFJzfB+rtg8ATplIkT6YrsUP/y6l2qJF
N5YTwQ0yUDi5Qceb6MvifEMXeMW2x8yDxNZSoe2FfZclJlZBLes5AJLMdZk0ZjOw
Dqvt7ngycPzMfontVLC5L8xr7sZzYVza9KXFZ/DxnGwTB5umwAg6TrodRJzqVR8+
5nTFxsXmM25IGCm0DIovPExLvVFiEw0pUEDwsoWPmb8HsqexZsKE2Ghh3mOlZnUH
x547xHV4xsyHxhpXzlkT7g5tGMgJoFjCaKb6mGCkKrxCiyYikMLHS2nOd6CeZ8VN
vIE8BUGIeuv6gK1/Z6KCG6a6s8qblNR66S+0WNZcsDitqmAmJ2LAPGUMUU0kbn2N
Fn6D1RuIPdx5Bubj0F74nAkBQ4oyI2yvntE9heSwyXKwC0RZBMfrX5V5GwVIdvqv
oADztB9hfan/TBahIbIGTcDO5hdGQ924rKD753wnS2+MKEoonFYNX6lxq8BFeh0W
KCJ/1Jr7HhHM9k4BimHhTyegNJYD6eCHitAFR118dt0n1HN0hgDlD/iqJJJYSwcW
8C2dQ3GqgCbItFAp09zyrxc86DfDP0alBLL6pGayeLA1JPl0AVW3E00K5bKDyeWf
omvngRPo4NubQz6NVnZ66nrso+zVaobG+8yyBsree/uvzkNiF0HNqYDXMcHc6Nx/
3QOUZbmd7171EItG20z/+FUlYBOK/YZRr+lSTIEf/82tXT7hnFJGntXjT5cik/bW
Akx3R5qgVoug0RpVVWnZuoKGTxBSpZlLzgYCE0qvMmLWaDrNLysYJHd0J00KmtXt
1yJz0P+dBwqLpM+SyY8At30QMZTdeWeoYQuOXsbtcrmhG4bVAmj9jH+bU1Sa58em
F5LCkGvrVY4rdYpXISGTpxy2y7KMtBRGawcRpuSEG9RKae6joE3YDuXdCK7cuYJK
rvtH8t09ZWB5cC/O/ORVmoPBvtn8QkSEwQx+u909t/I=
=8oAQ
-----END PGP MESSAGE-----

--x2iAciceo0NrKsl4uFVFGTgJ6k0ocg8FW--

