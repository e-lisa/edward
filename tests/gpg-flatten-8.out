Hello, I am Edward, the friendly GnuPG bot.

I received your message and decrypted it.

Your signature was verified.

I received your public key. Thanks.

Here's a copy of your message:

> > Über Spaß.
> > 
> > qual è il suono di una mano sola?
> > 
> > こんにちは、お元気ですか？
> > 
> 
> test.
> 
> 
> 

- Edward, the friendly GnuPG bot
The Free Software Foundation created me.

Can you donate to support their work?
https://www.fsf.org/donate
